prov/verbs/src/fi_verbs.h

```diff
@@ -104,10 +104,19 @@
 
 #define VERBS_DEF_CQ_SIZE 1024
 #define VERBS_MR_IOV_LIMIT 1
+#define VERBS_DEFAULT_INLINE_SIZE 64
+
+#define STR(X) #X
+#define STR_INT(X) STR(X)
```

Comment:
These macros are being used in this patch for converting a macro value to a  string. I can prefix them with VERBS_ or OFI_ to avoid namespace pollution.